test/parallel/test-regression-object-prototype.js

```diff
@@ -1,7 +0,0 @@
-'use strict';
```

Comment:
@silverwind Actually, I learnt something new when I was debugging this. When we add something to `Object.prototype`, it is getting added to the `global` as well.

```
> process.versions
{ http_parser: '2.5.0',
  node: '5.0.0-pre',
  v8: '4.5.103.30',
  uv: '1.7.3',
  zlib: '1.2.8',
  ares: '1.10.1-DEV',
  modules: '46',
  openssl: '1.0.2d' }
> Object.prototype.silverwind = 1;
1
> global.silverwind
1
```

So, the property `xadsadsdasasdxx` is leaked to global. But, this doesn't have any reference to why this is necessary. So I thought that it may not be necessary to keep it around.